.github/actions/asana-log-message/action.yml

```diff
@@ -24,10 +24,10 @@ runs:
   using: "composite"
   steps:
     - id: get-automation-subtask
-      uses: ./.github/actions/asana-get-release-automation-subtask-id
-      with:
-        access-token: ${{ inputs.access-token }}
-        task-url: ${{ inputs.task-url }}
+      shell: bash
```

Comment:
is this needed?

Reply:
Yes, shell definition is needed for actions steps.